src/sphinx_antsibull_ext/antsibull-minimal.css.license

```diff
@@ -0,0 +1,3 @@
+GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+SPDX-FileCopyrightText: Ansible Project
+SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Is this supposed to be included both here and in the file?

Reply:
Kind of. `SPDX-License-Identifier` is not a duplicate; removing `SPDX-FileCopyrightText` results in linting failures (the copyright text is no longer found). The only thing that's something we can remove here is the first line, but for consistency I would keep it.

Reply:
Resolve